tests/index.html

```diff
@@ -22,13 +22,13 @@
     {{content-for "test-body"}}
 
     <div id="qunit"></div>
-    <div id="qunit-fixture"></div>
-
-    <div id="ember-testing-container">
-      <div id="ember-testing"></div>
+    <div id="qunit-fixture">
+      <div id="ember-testing-container">
+        <div id="ember-testing"></div>
+      </div>
     </div>
 
-    <script src="/testem.js" integrity=""></script>
+    <script src="/testem.js" integrity="" data-embroider-ignore></script>
```

Comment:
What does this do?

Reply:
This ensures embroider doesn't warn that it can't find this, as this is a dynamic files provided by a middleware which shouldn't and cannot be bundled.